definitions.yml

```diff
@@ -0,0 +1,6 @@
+:sles12:
+  :yast_url: "http://download.suse.de/ibs/Devel:/YaST:/Head/SLE-12-SP1/"
+  :iso_url: "http://dist.suse.de/install/SLE-12-Server-GM/SLE-12-Server-DVD-x86_64-GM-DVD1.iso"
+:sles12-sp1:
+  :yast_url: "http://download.suse.de/ibs/Devel:/YaST:/Head/SLE-12-SP1/"
```

Comment:
BTW, we might remove the /Head/ as discussed on the planning meeting today, @jreidinger can you comment?